packages/multi-test/src/wasm.rs

```diff
@@ -389,28 +389,26 @@ where
         });
 
         // call reply if meaningful
-        if let Ok(r) = res {
+        if let Ok(mut r) = res {
```

Comment:
This if-let implementation was actually root cause of the problem. Previously when there was a submsg response received, but no reply was called, replay was returned including data received from submsg execution. It does not make sense, as submsg execution result is not propagated directly, it is only send to reply handled and forget, those in case when there is response from submsg, but no reply is called, `Ok(None)` is now returned here.